Scripts/Modules/SerializedPreferencesModule.cs

```diff
@@ -94,8 +95,10 @@ internal string SerializePreferences()
 				}
 
 				if (item != null)
-					preferences.items.Add(item);
+					payloads[serializer.GetType()] = item;
 			}
+
+			preferences.items.AddRange(payloads.Values);
```

Comment:
This is again to protect against adding an item of the same type more than once. The dictionary is ensuring that we just overwrite payloads of the same type instead of holding on to them.